services/webhook/matrix.go

```diff
@@ -212,7 +212,7 @@ func (m *MatrixPayload) Repository(p *api.RepositoryPayload) (api.Payloader, err
 
 func (m *MatrixPayload) Package(p *api.PackagePayload) (api.Payloader, error) {
 	senderLink := MatrixLinkFormatter(setting.AppURL+p.Sender.UserName, p.Sender.UserName)
-	repoLink := MatrixLinkFormatter(p.Repository.HTMLURL, p.Repository.FullName)
+	repoLink := MatrixLinkFormatter(p.Package.HTMLURL, p.Package.Name)
```

Comment:
```suggestion
	packageLink := MatrixLinkFormatter(p.Package.HTMLURL, p.Package.Name)
```
maybe the variable name should also be changed.


ps: `p.Repository` is always `null`, as `api.PackagePayload` comes from here:
https://github.com/go-gitea/gitea/blob/fce1d5d7dce6a16429d0fe043555eac2c083ae7b/services/webhook/notifier.go#L882-L886
And as the comment mentioned here, `Repository` can be `null` when user upload/delete a package to an organization. So we should not use `p.Repository` here.
https://github.com/go-gitea/gitea/blob/fce1d5d7dce6a16429d0fe043555eac2c083ae7b/services/actions/notifier_helper.go#L310-L315